src/main/scheduler/scheduler_tasks.c

```diff
@@ -42,7 +42,14 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .taskName = "GYRO",
         .taskFunc = taskGyro,
         .desiredPeriod = 1000000 / 1000, //Run at 1000Hz
-        .staticPriority = TASK_PRIORITY_REALTIME
+        .staticPriority = TASK_PRIORITY_REALTIME,
+    },
+
+    [TASK_ACC] = {
+        .taskName = "ACC",
+        .taskFunc = taskAcc,
+        .desiredPeriod = 1000000 / 120, //Run at 120Hz, with LPF at 15Hz this is even too much
```

Comment:
Acro-only this should be fine, ANGLE might (and probably) will drift. Navigation will also be heavily affected. We shouldn't break ANGLE/Navigation while chasing performance, however for R&D I figure this is fine :smiley: